spec/models/user_spec.rb

```diff
@@ -12,7 +12,7 @@
   end
 
   describe "enum" do
-    it { should define_enum_for(:role).with([:default, :merchant, :admin]) }
+    it { should define_enum_for(:role).with_values([:default, :merchant, :admin]) }
```

Comment:
This fixes one of the non-error error messages that pops up when we run RSpec so now it is real smooth.

Reply:
I like that.